README.md

```diff
@@ -1,6 +1,6 @@
 # Checkmk's Grafana Data Source Plugin
 
-![CI](https://github.com/checmk/grafana-checkmk-datasource/actions/workflows/ci.yml/badge.svg)
+![CI](https://github.com/Checkmk/grafana-checkmk-datasource/actions/workflows/ci.yml/badge.svg)
```

Comment:
Ok, I'll have to rewrite these everywhere else as well. I'll put it on my TODO list.

Reply:
are you sure? i can not finy any matches for this typo.

Reply:
Found a few other ones in the plugin.json. I'll do a fixup MR